cre.cpp

```diff
@@ -394,6 +394,12 @@ static int setStringProperty(lua_State *L) {
     return 0;
 }
 
+static int getGammaLevel(lua_State *L) {
+	lua_pushnumber(L, fontMan->GetGamma());
```

Comment:
Pssht, these are tabs. I realize the document is a mixture but I'd say to stick with our style guide for new stuff. ;-)

Reply:
They were tabs in the 2 functions related to Gamma just below - just cut and pasted from them.